app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -162,9 +163,12 @@ class AuthenticationFragment : Fragment() {
     }
 
     private fun buildAuthUrl(base: String): String {
+        val url = base.toHttpUrl()
```

Comment:
`toHttpUrl()` will throw an exception if not valid so better to move it inside the `try`/`catch` block